docs/content/en/docs/workshops/packages/adot/adot_amp_amg.md

```diff
@@ -0,0 +1,612 @@
+
+### IRSA Set Up Test
+To ensure IRSA is set up properly in the cluster, a user can create an `awscli` pod for testing.
+1. Apply the following yaml file in the cluster:
+    ```bash
+    kubectl apply -f - <<EOF
+    apiVersion: v1
+    kind: Pod
+    metadata:
+      name: awscli
+    spec:
+      serviceAccountName: pod-identity-webhook
+      containers:
+      - image: amazon/aws-cli
+        command:
+          - sleep
+          - "infinity"
+        name: awscli
+        resources: {}
+      dnsPolicy: ClusterFirst
+      restartPolicy: Always
+    EOF
+    ```
+
+1. Exec into the pod:
+    ```
+    kubectl exec -it awscli -- /bin/bash
+    ```
+
+1. Check if the pod can list AMP workspaces:
+    ```
+    aws amp list-workspaces --region=us-west-2
+    ```
+
+1. If the pod has issues listing AMP workspaces, re-visit IRSA set up guidance before proceeding to the next step.
+
+1. Exit the pod:
+    ```
+    exit
+    ```
+
+## Install EKS-A ADOT package
+
+The EKS-A ADOT package will be created with three components:
+
+1. the Prometheus Receiver, which is designed to be a drop-in replacement for a Prometheus Server and is capable of scraping metrics from microservices instrumented with the [Prometheus client library](https://prometheus.io/docs/instrumenting/clientlibs/);
+
+1. the Prometheus Remote Write Exporter, which employs the remote write features and send metrics to AMP for long term storage;
+
+1. the Sigv4 Authentication Extension, which enables ADOT pods to authenticate to AWS services.
+
+Follow steps below to complete the ADOT package installation:
+
+1. Update the following config file. Review comments carefully and replace everything that is wrapped with a `<>` tag. Note this configuration aims to mimic the Prometheus community helm chart. A user can tailor the scrap targets further by modifying the receiver section below. Refer to [ADOT package spec]({{< relref "../../../reference/packagespec/adot" >}}) for additional explanations of each section.
+
+    <details>
+      <summary>Click to expand ADOT package config</summary>
+      
+      ```yaml
+      apiVersion: packages.eks.amazonaws.com/v1alpha1
+      kind: Package
+      metadata:
+        name: my-adot
+        namespace: eksa-packages
+      spec:
+        packageName: adot
+        targetNamespace: default # this needs to match the namespace of the serviceAccount below
+        config: |
+          mode: deployment
+
+          serviceAccount:
+            # Specifies whether a service account should be created
+            create: false
+            # Annotations to add to the service account
+            annotations: {}
+            # Specifies the serviceAccount annotated with eks.amazonaws.com/role-arn.
+            name: "pod-identity-webhook" # name of the service account created at step Create an EKS-A cluster with IRSA
```

Comment:
Change "Create an EKS-A cluster with IRSA" to however you modify the heading.

Reply:
Updated